Gulpfile.mjs

```diff
@@ -384,6 +384,7 @@ function buildRollup(packages, buildStandalone) {
                 ) {
                   return;
                 }
+                break;
```

Comment:
It seems to me we should have emitted the warning here, otherwise it makes no sense that we specifically ignore warnings from polyfill-provider.

Reply:
It was probably just a leftover, because deleting this line doesn't change the warnings currently present in our build.

```suggestion
```